MapboxMobileEvents/MMEAPIClient.m

```diff
@@ -54,18 +53,60 @@ - (void)postEvent:(MMEEvent *)event completionHandler:(nullable void (^)(NSError
     [self postEvents:@[event] completionHandler:completionHandler];
 }
 
+- (void)getConfigurationWithCompletionHandler:(nullable void (^)(NSError * _Nullable error, NSData * _Nullable data))completionHandler {
+    NSURLRequest *request = [self requestForConfiguration];
+    
+    [self.sessionWrapper processRequest:request completionHandler:^(NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error) {
+        if ([response isKindOfClass:[NSHTTPURLResponse class]]) {
+            NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *)response;
+            NSError *statusError = [self statusErrorFromRequest:request andHTTPResponse:httpResponse];
+            if (completionHandler) {
+                error = error ?: statusError;
+                completionHandler(error, data);
+            }
+        }
```

Comment:
We need to call the completion handler in all cases, so we need a `else` to this clause.

Reply:
Noting that this has been addressed.